week5/serving/ml_decision_tree_sample/cleaner.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+def remove_model_from_local_directory(model_name: str):
+    if os.path.exists(model_name):
+        os.remove(model_name)
+        print(f"The file {model_name} has been deleted successfully")
+    else:
+        print(f"The file {model_name} does not exist!")
+
+
+def remove_wand_local_directory():
+    try:
+        os.system("rm -rf wandb/")
```

Comment:
don't use `os.system("rm -rf wandb/")`

please use: `shutil.rmtree`

Reply:
did it